src/applications/vaos/services/mocks/index.js

```diff
@@ -176,6 +176,8 @@ const responses = {
         attributes: {
           ...requestAttributes,
           status: 'Cancelled',
+          appointmentRequestDetailCode: ['DETCODE8'],
+          cancelationReason: { coding: [{ code: 'pat' }] },
```

Comment:
Minor, but there is a constant, CANCELLATION_REASONS.patient that can be used for 'pat'.